metadata/service/aggregator/single.yml

```diff
@@ -6,3 +6,4 @@ parameters:
   heka:
     aggregator:
       enabled: true
+      timezone: ${_param:linux_timezone}
```

Comment:
why not providing a "sane" default here. It will explicitly document that this parameter should/must be overridden bu user data model :

```
 parameters:
  _params:
     linux_timezone: UTC
```